types/dialogflow/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for dialogflow 0.9
+// Type definitions for dialogflow 0.11
 // Project: https://github.com/googleapis/nodejs-dialogflow
 // Definitions by: Daniel Dyla <https://github.com/dyladan>
 //                 Tom Carrio <https://github.com/tcarrio>
 //                 Khalil Choudhry <https://github.com/khalilchoudhry>
+//                 Joseph Thibeault <https://github.com/jrthib>
```

Comment:
Only include yourself if you are dedicating yourself to maintaining this package. This will include code reviews on all future pull requests.

Reply:
Makes sense. I will PR to remove myself when I'm no longer able.